.github/workflows/checks.yaml

```diff
@@ -0,0 +1,15 @@
+on: [pull_request]
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: "shivammathur/setup-php@v2"
+              with:
+                php-version: "latest"
+            - uses: actions/checkout@v4
+            - uses: ramsey/composer-install@v3
+            - uses: ramsey/composer-install@v3
```

Comment:
Why does virtue-jwt require additional installation? Shouldn't main composer.json contain all dependencies? I believe it is needed for some reason, but could you please share it with us? 😄

Reply:
we acctually don't